### PR TITLE
Remove discover_nodes/python-dracclient

### PR DESCRIPTION
**Pull request by Rachit7194**
@cdearborn @rpioso  @digambar15

### CODE REVIEW

File: src/pilot/is_idrac_ready.py [hunk 1]
```diff
@@ -66,7 +60,7 @@ def main():
         drac_user = node["pm_user"]
         drac_password = node["pm_password"]
 
-        drac_client = DRACClient(drac_ip, drac_user, drac_password)
+        drac_client = client.DRACClient(drac_ip, drac_user, drac_password)
```
Comment (cdearborn) [inline src/pilot/is_idrac_ready.py]: I think you need to import client

File: src/pilot/assign_role.py [hunk 2]
```diff
@@ -125,7 +121,7 @@ def parse_arguments():
                         metavar="ROLE")
     parser.add_argument("-f",
                         "--flavor-settings",
-                        default="~/pilot/flavors_settings.json",
+                        default="flavors_settings.json",
```
Comment (cdearborn) [inline src/pilot/assign_role.py]: I don't think we should be changing the defaults or behavior for flavor_settings.json as part of this patch

Comment (Rachit7194) [reply src/pilot/assign_role.py]: Okay. Replace as is

**Comment by Rachit7194** [comment]
Hi Chris, deployment on R7 stamp completed with this change. Now all the testing completed with this PR.

**Pull request merged**